yarn-project/aztec-node/src/aztec-node/server.ts

```diff
@@ -285,6 +285,7 @@ export class AztecNodeService implements AztecNode {
     await this.p2pClient.stop();
     await this.worldStateSynchronizer.stop();
     await this.blockSource.stop();
+    await this.merkleTreesDb.close();
```

Comment:
In the context of a jest test, old references to nodes/pxe stay and they need to clean up all of the resources that they use. LevelDB can only have one reader/one writer per db instance